internal/events/tailer_test.go

```
@@ -0,0 +1,147 @@
+
+	ch := make(chan Event, 8)
+	invalid := make(chan string, 8)
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		Tail(ctx, path, ch, invalid)
+	}()
+
+	f, _ := os.OpenFile(path, os.O_APPEND|os.O_WRONLY, 0o644)
+	defer f.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `f.Close` is not checked

(errcheck)